test/Microsoft.AspNet.Authentication.Test/Cookies/CookieMiddlewareTests.cs

```diff
@@ -748,12 +749,14 @@ private Task SignOutAsWrong(HttpContext context)
         [Fact]
         public async Task MapWillNotAffectChallenge()
         {
-            var server = TestServer.Create(app =>
+            var builder = new WebApplicationBuilder()
```

Comment:
I know it's not specific to the security tests, but this new syntax is a bit cumbersome.
Curious: why are the static `Create` methods removed?